console/src/domain/runtime/components/RuntimeTreeSelector.tsx

```diff
@@ -19,12 +19,12 @@ const RuntimeTreeNode = themedStyled('div', () => ({
 
 export function RuntimeTreeSelector(props: any) {
     const [t] = useTranslation()
-    const { projectId } = props
+    const { projectId, builtIn } = props
     const runtimeInfo = useFetchRuntimeTree(projectId)
     const $treeData = React.useMemo(() => {
-        if (!runtimeInfo.isSuccess) return []
-
-        const treeData: TreeNodeData[] = runtimeInfo.data.map((runtime) => {
+        if (!runtimeInfo.isSuccess) return builtIn? [builtIn] : []
+        let mergedItems = builtIn ? [builtIn, ...runtimeInfo.data.filter(i => i.versions[0].id !== builtIn.versions[0].id)] : runtimeInfo.data
```

Comment:
The corresponding version may not be listed first?

Reply:
yes, fixed